torch/fx/examples/inline_function.py

```diff
@@ -0,0 +1,68 @@
+import torch
+from torch.fx import Proxy, symbolic_trace
+from torch.fx.node import map_arg
+
+
+'''
+How to inline a function into an existing Graph
+
+If you're using the default Tracer to symbolically trace, ``torch.nn``
```

Comment:
Probably want to make it clear this is only an example scenario. The code is good for any situation in which one wants to inline a call